utils/docker/Dockerfile.centos.7

```
@@ -78,6 +78,9 @@ RUN yum -y install \
            java-1.8.0-openjdk python-distro python36-distro
 RUN yum -y install openmpi3-devel
 
+# libunwind for Argobots
+RUN yum -y install libunwind libunwind-devel
```

```suggestion
RUN yum -y install libunwind-devel
```

`libunwind-devel` will depend on `libunwind`.

---

Well, this is harmless, and thus why not be as explicit as possible since both are really needed respectively at run and build times ??

---

It's not harmless.  The relationship between `libunwind-devel` and the other packages (i.e. `libunwind`) is at the sole discretion of the packager.  There is no guarantee that `libunwind` will always be the package that `libunwind-devel` depends on.  Depending on (only) `libunwind-devel` is the only assumption you can be assured of now and in the future, and the packager is then free to change the name, composition, or any other aspect of the `libunwind*` packages without impacting what consumers of `libunwind-devel` might be assuming.

TL; DR: Over-specifying dependencies can have all sorts of problems in the future that correctly specifying only the minimum dependency set prevents.

Indeed, we are addressing a bug in another package at this current time due to over-specifying dependencies.

---

Ok...